src/main.ts

```diff
@@ -108,6 +108,9 @@ export function run(
           `codeclimate.${i}.json`
         ];
         if (codeClimateDebug === 'true') commands.push('--debug');
+        if (!coveragePrefix && typeof coveragePrefix === 'string') {
```

Comment:
@paambaati - As far as I can tell this logic is only going to append `--prefix` when `coveragePrefix` is `""` (empty string). I'd make a PR to address/fix but I'm not quite sure what the goal here is.  Are you trying to only add `--prefix` only when `coveragePrefix` was passed?

Reply:
Specifically, I'm seeing the following command being executed:

```js
/home/runner/work/ember-cli/ember-cli/cc-reporter format-coverage coverage/lcov.info -t lcov -o codeclimate.0.json --prefix 
```

When the configuration is:

```yml
      - name: Test && Report to Code Climate
        uses: paambaati/codeclimate-action@v2.5.4
        with:
          coverageCommand: yarn test:cover
          coverageLocations: "coverage/lcov.info:lcov"
```

Reply:
> Are you trying to only add --prefix only when coveragePrefix was passed?

That is correct.

> I'd make a PR to address/fix

Much appreciated, please and thank you.

Reply:
No problem! Here is what I was thinking https://github.com/paambaati/codeclimate-action/pull/131 for the solution.